src/components/Hashcat/Hashcat.tsx

```diff
@@ -41,8 +18,22 @@ interface FormValuesType {
     maxPwdLen: number;
 }
 
-const inputType = ["Hash Value", "File"];
-const attackMode = ["Straight", "Brute-force", "Hybride Wordlist + Mask"];
+const title = "Hashcat";
+const description_userguide = `
```

Comment:
It should be description
```suggestion
const description = `
```